benefits/core/templates/core/help.html

```diff
@@ -3,65 +3,137 @@
+            We partner with local transit agencies, the California Department of Technology, and Login.gov to confirm the
+            identity of riders who are eligible to receive transit benefits. Once we are able to confirm a rider's identity and
+            eligibility, our payment partner Littlepay helps us connect a transit benefit to the rider's contactless card of
+            choice.
+          {% endblocktranslate %}
+        </p>
 
-        <h2 class="pt-8" id="payment-options">{% translate "core.pages.help.payment_options" %}</h2>
-        <p class="pt-4">{% translate "eligibility.pages.start.modal.p[0]" %}</p>
-        <p class="pt-4">{% translate "eligibility.pages.start.modal.p[1]" %}</p>
+        <h2 class="pt-8" id="payment-options">{% translate "What kind of credit card can I use?" %}</h2>
+        <p class="pt-4">
+          {% blocktranslate trimmed %}
+            Benefits only works with contactless debit or credit cards. We do not currently support mobile wallets or digital
+            cards, such as Apple Pay, Google Wallet, or Samsung Wallet.
+          {% endblocktranslate %}
+        </p>
+        <p class="pt-4">{% translate "The contactless symbol is four curved lines, like this:" %}</p>
         <p class="contactless-symbol pt-3">
           <img class="icon mx-auto d-block"
                width="40"
                height="50"
                src="{% static 'img/icon/contactless.svg' %}"
-               alt="{% translate "core.icons.contactless" context "image alt text" %}" />
+               alt="{% translate "Four curved lines on contactless-enabled cards" context "image alt text" %}" />
+        </p>
+        <p class="pt-4">{% translate "Your card must include a Visa or Mastercard logo." %}</p>
+        <p class="pt-4">
+          {% blocktranslate trimmed %}
+            Don’t have access to a contactless card? You can request a contactless card from your bank, or one of the companies
+            that offer free contactless prepaid debit cards, such as the Cash App Visa debit card or the Venmo Mastercard debit
+            card.
+          {% endblocktranslate %}
         </p>
-        <p class="pt-4">{% translate "eligibility.pages.start.modal.p[2]" %}</p>
-        <p class="pt-4">{% translate "eligibility.pages.start.modal.p[3]" %}</p>
         {% if agency %}
           <p class="pt-4">
-            {% blocktranslate with short_name=agency.short_name website=agency.info_url %}eligibility.pages.start.modal.p[4]{{ short_name }}{{ website }}{% endblocktranslate %}
+            {% blocktranslate trimmed with short_name=agency.short_name website=agency.info_url %}
+              You can still get your transit benefit by going through {{ short_name }}'s application process. For updates on
+              additional options, please check back on this website, or
+              <a href="{{ website }}" target="_blank" rel="noopener noreferrer">contact {{ short_name }}</a>."
```

Comment:
Much better to have the string here, now the link is just a normal `<a>` element in HTML

Reply:
Remove the last quotation mark <img width="872" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/36c9c712-7864-4ab2-ae4e-b6009739e591">

Reply:
fix confirmed: 
<img width="915" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/e9225361-3f85-48da-9c0b-b57c1c251898">
<img width="889" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/7c646941-1724-4fb4-a3e1-b8057f21587a">